compiler/src/iree/compiler/Codegen/LLVMGPU/test/nvvm_pipeline_test.mlir

```diff
@@ -1,905 +0,0 @@
-// RUN: iree-opt --split-input-file --iree-gpu-test-target=sm_60 --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(builtin.module(iree-codegen-llvmgpu-configuration-pipeline), iree-codegen-linalg-to-nvvm-pipeline)))" -iree-codegen-llvmgpu-use-wmma %s | FileCheck %s
```

Comment:
We shouldn't delete everything here, only the matmul tests. This includes some tests that go down the simpler/default pipeline that I keep updated as necessary and we don't want to drop support for.